plugins/org.eclipse.n4js.ts.model/model/Types.xcore

```diff
@@ -1011,6 +1024,9 @@ abstract class TMember extends IdentifiableElement, TAnnotableElement, SyntaxRel
 
 	boolean hasComputedName
 
+	/** Tells if this member is a composed member created by ComposedMemberFactory. */
```

Comment:
a link won't work, because the .ts bundle does not have a dependency to the bundle containing `ComposedMemberFactory`.